orca-echo/src/main/groovy/com/netflix/spinnaker/orca/echo/spring/EchoNotifyingStageListener.groovy

```diff
@@ -127,12 +126,12 @@ class EchoNotifyingStageListener implements StageListener {
       }
 
       try {
-        MDC.put(SPINNAKER_EXECUTION_ID, stage.execution.id);
-        MDC.put(SPINNAKER_USER, stage.execution?.authentication?.user ?: "anonymous")
+        MDC.put(AuthenticatedRequest.SpinnakerHeader.EXECUTION_ID.header, stage.execution.id)
+        MDC.put(AuthenticatedRequest.SpinnakerHeader.USER.header, stage.execution?.authentication?.user ?: "anonymous")
```

Comment:
Unrelated to this PR ... but sure would be nice to swap that `stage.execution.authentication.user` out for the more appropriate `AuthenticationAware` logic (ie. look at ancestors for a `AuthenticatedStage` stage). 

I don't believe there's anything in `echo` that actually needs the correct authenticated user which would explain why we haven't run into an issue with this previously.

This PR looks good otherwise.

Reply:
thanks. Let me look into it. Also, will pull in new kork now and update this PR